website/docs/command-reference/catalog.md

```diff
@@ -0,0 +1,119 @@
+
+## Services
+
+odo can help you deploy *services* with the help of *operators*.
+
+### Listing services
+
+You can get the list of available operators and their associate services with the command:
+
+```
+$ odo catalog list services
+Services available through Operators
+NAME                                 CRDs
+postgresql-operator.v0.1.1           Backup, Database
+redis-operator.v0.8.0                RedisCluster, Redis
+```
+
+In this example, you can see that two operators are installed in your cluster. The first one to deploy PostgreSQL related services, the second to deploy Redis related services. The PostgreSQL operator offers two services: `Backup` and `Database`.
+
+> Note that only operators deployed with the help of the [*Operator Lifecycle Manager*](https://olm.operatorframework.io/) are supported by odo. See [Installing the Operator Lifecycle Manager (OLM)](/docs/getting-started/cluster-setup/kubernetes#installing-the-operator-lifecycle-manager-olm) for more information.
+
+> Also note that odo fetches the `ClusterServiceVersion` (`CSV`) resources of the current namespace in a *Succeeded* phase to get the list of available operators. When a new namespace is created, these resources are automatically added to the namespace, and some time is necessary for them to be in the *Succeeded* phase. So, please be patient when you try to list the services from a newly created namespace, you could get an empty list for some time.
+
+### Searching services
+
+You can search for specific services by keyword with the command:
+
+```
+$ odo catalog search service postgre
+Services available through Operators
+NAME                           CRDs
+postgresql-operator.v0.1.1     Backup, Database
+```
+
+A similar list will be displayed, containing only the relevant operators, whose name contains the searched keyword.
+
+### Getting information about a service
+
+You can get more information about a specific service with the command:
+
+```
+$ odo catalog describe service postgresql-operator.v0.1.1/Database
+KIND:    Database
+VERSION: v1alpha1
+
+DESCRIPTION:
+     Database is the Schema for the the Database Database API
+
+FIELDS:
+   awsAccessKeyId (string)   
+     AWS S3 accessKey/token ID
+
+     Key ID of AWS S3 storage. Default Value: nil Required to create the Secret
+     with the data to allow send the backup files to AWS S3 storage.
+[...]
+```
+
+A service is represented in your cluster by a `CustomResourceDefinition` (commonly named `CRD`). This command will display the details about this CRD: its kind, its version, and the list of fields available to define an instance of this custom resource.
+
+The list of fields is extracted from the *OpenAPI schema* included in the `CRD`. As this information is optional in a `CRD`, if it is not present, the fields information are extracted from the `ClusterServiceVersion` (`CSV`) representing the service instead.
```

Comment:
The list of fields is extracted from the *OpenAPI schema* included in the `CRD`. This information is optional in a `CRD`, and if it is not present, it is extracted from the `ClusterServiceVersion` (`CSV`) representing the service instead.

Reply:
The part about information being extracted from CSV is unclear to me, but I am not sure if this is the right place or if we even need to add an example to elaborate more on it here.

Reply:
This is more for reference, for people knowing what we are talking about (to be sure we agree on which order we get the information: first CRD, else CSV)